### PR TITLE
Coop logo

### PR DESCRIPTION
**Pull request by martabelfm**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

Coop sent me this logo.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by tauthomas01** [comment]
Hi @martabelfm,

It's missing the border around the icon. An example can be found inside the [contribution document](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

**Comment by martabelfm** [comment]
> Hi @martabelfm,
> 
> It's missing the border around the icon. An example can be found inside the [contribution document](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

I am sorry, very unfamiliar with SVG-s. It seems to have a border around the icon when looking at it, am I missing something?

**Comment by tauthomas01** [comment]
@martabelfm What I mean is the border should follow the same style as the contribution document.

See this [video](https://screenshot.click/23-03-lh5cl-pubcc.mp4) for example.

**Comment by martabelfm** [comment]
> @martabelfm What I mean is the border should follow the same style as the contribution document.
> 
> See this [video](https://screenshot.click/23-03-lh5cl-pubcc.mp4) for example.

Okay, I did not see the black border when I looked at it myself. Tried with chrome, firefox and some online editor. Anyway changed it to match the visa file.

File: app/assets/images/payment_icons/coop.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" aria-labelledby="pi-coop"><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M24.7 4.7c-3.3 0-5.9 1.2-6.8 3.3-.3.7-.6 1.5-.6 5.9v1.8l-1.8-2c-.6.2-1.3.3-2.1.3-2.4 0-3.3-1.5-3.3-3.3 0-1.8.9-3.3 3.3-3.3 1.2 0 2.1.6 2.7 1.2l1.8-3c-1.2-.6-2.7-.9-4.5-.9-4.7 0-7.7 2.7-7.7 6.2 0 3.6 2.7 6.2 7.7 6.2 1.6 0 2.9-.3 3.9-.7v2.8h3.9v-2.7c.9.3 2.1.6 3.3.6 4.7 0 7.7-2.7 7.7-6.2.2-3.6-2.7-6.2-7.5-6.2zm0 9.2c-.6 0-.9 0-1.5-.3l-1.8 2.1v-5.1c0-1.8.9-3.3 3.3-3.3S28 8.8 28 10.6c0 1.8-.9 3.3-3.3 3.3z" fill="#0078d2"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/coop.svg:1]: We need to add `role="img"` and `viewBox="0 0 38 24"`

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-coop"><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M24.7 4.7c-3.3 0-5.9 1.2-6.8 3.3-.3.7-.6 1.5-.6 5.9v1.8l-1.8-2c-.6.2-1.3.3-2.1.3-2.4 0-3.3-1.5-3.3-3.3 0-1.8.9-3.3 3.3-3.3 1.2 0 2.1.6 2.7 1.2l1.8-3c-1.2-.6-2.7-.9-4.5-.9-4.7 0-7.7 2.7-7.7 6.2 0 3.6 2.7 6.2 7.7 6.2 1.6 0 2.9-.3 3.9-.7v2.8h3.9v-2.7c.9.3 2.1.6 3.3.6 4.7 0 7.7-2.7 7.7-6.2.2-3.6-2.7-6.2-7.5-6.2zm0 9.2c-.6 0-.9 0-1.5-.3l-1.8 2.1v-5.1c0-1.8.9-3.3 3.3-3.3S28 8.8 28 10.6c0 1.8-.9 3.3-3.3 3.3z" fill="#0078d2"/></svg>
```

**Comment by martabelfm** [comment]
> One more thing and the rest looks good.

Anything else?

**Comment by tauthomas01** [comment]
It's missing <title> attribute.

**Comment by martabelfm** [comment]
> It's missing `<title>` attribute.

Lets try one more time.

**Comment by martabelfm** [comment]
> Looks good! Can you rebase the branch off master and fix the conflict?

done

**Comment by tauthomas01** [comment]
Looking good, thank you for the changes.

**Pull request merged**